### ISSUE TITLE
Faeries won't spawn

### ISSUE THREAD

**Issue by faeldray** [BenjaminSutter/genera#18]
Genera version 1.0.7
Forge version 2611

I started a new world and planted some nightshade in a forest biome but no faeries are spawning. I went back and downgraded to 1.0.6 and had no issues with faeries spawning. I don't see any sort of errors in the log so I'm not sure what's going wrong.

**Comment by BenjaminSutter** [comment]
The underlying mechanism that spawns Faeries was changed quite a bit in 1.0.7 I noticed that the nightshade spawned them at a slower rate, but they were spawning for me in testing. I'll look into it. Thanks.

**Comment by faeldray** [comment]
At what rate should I be expecting faeries to spawn in 1.0.7? I had a field of about a dozen nightshade (just to test) and after 5 minutes, I still wasn't seeing any.

**Comment by BenjaminSutter** [comment]
Yeah, that's definitely too slow. I got some after two or so minutes. I'm not home yet, but I will be soon. I'm on a trip until Sunday, so I may have a little time to fix this, but I might not get it done until Sunday.

**Comment by faeldray** [comment]
That's understandable. :)

If it helps any, I created a new modpack to test that only contained Genera and Forge 2611 to make sure the issue wasn't a mod conflict. I also tried switching to Forge 2619 and 2587 to see if that made a difference. Unfortunately there still aren't any faeries spawning in any of the situations.

**Comment by BenjaminSutter** [comment]
Thanks for putting the effort in. I'll fix it as soon as I can.

**Comment by faeldray** [comment]
Thanks for the update!

**Comment by faeldray** [comment]
You mentioned other flowers...are burdocks suppose to spawn faeries as well? Because I did another test with just Genera and Forge 2611 and I couldn't get faeries to spawn that way either (after waiting 10 minutes).

![2018-03-18_17 47 52](https://user-images.githubusercontent.com/967709/37572138-bffdff86-2ad4-11e8-96e9-e11b2b88243a.png)

**Comment by BenjaminSutter** [comment]
Really? That's odd. They spawn pretty regularly for me. I'll look into it. Thanks for all the testing!

**Comment by faeldray** [comment]
I feel like I'm doing something wrong here since you're not getting the same results as me, but I can't figure out what. :P

I don't know if this could have anything to do with it but I checked the config/genera folder and it contains /faerie and /rituals but they're both empty. Is something suppose to be in there?

**Comment by BenjaminSutter** [comment]
No, those are custom user-provided Faeries and rituals. Nothing should be in them by default. If the plant is fully grown (stage 2), then should spawn. It's weird.

**Comment by DefinitelyNotFred** [comment]
I'm having the same issue. I've planted over a hundred nightshade plants with Genera-1.12.2-1.0.7 on a clean install of minecraft 1.12.2 and forge 1.12.2-14.23.2.2611 with no faerie spawns. I've tried switching to the Twitch client, differing light levels, waiting long periods of time, and not using bonemeal. After all that, I downgraded to the 1.12.2-1.0.6 version and went back to my Nightshade farm world. I got a message saying that the genera:faerie_registry was missing in this save and that continuing would remove that registry. Once I got in, I found enough Faeries in my Nightshade farm to completely fill the hotbar in a few seconds.

I'm also having rendering issues with the Faeries which may be related but I think I'll make a new issue for that.

**Comment by BenjaminSutter** [comment]
The rendering issues are known. Working on them. I don't know what's wrong with spawning, but I think it'll likely be fixed with the next update. I've cleaned a lot.

**Comment by DefinitelyNotFred** [comment]
I know you know there are rendering issues, but from what I've read they seem pretty basic. As far as I can tell, what I'm seeing hasn't been addressed:
![ezgif com-optimize](https://user-images.githubusercontent.com/30205205/38160079-a9362e20-347c-11e8-81e8-a8ded5049b5d.gif)
As you can see, there is a lot of stuttering movement with the faeries and there is the fact of how they show enmasse when I'm at a certain distance but when I'm closer they almost completely vanish.

**Comment by DefinitelyNotFred** [comment]
I should mention that my graphics card is old (AMD Radeon HD 6900 Series) and the rest of my computer ain't the greatest either.
